octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -589,7 +595,7 @@ def clean_recent_trade(self, recent_trade):
             recent_trade.pop(ecoc.ORDER.value)
             recent_trade.pop(ecoc.FEE.value)
             recent_trade.pop(ecoc.TYPE.value)
-            recent_trade.pop(ecoc.TAKERORMAKER.value)
+            recent_trade.pop(ecoc.TAKER_OR_MAKER.value)
```

Comment:
If we changed this value in enums, I guess this will now work anymore (ccxt is in camelcase)

Reply:
:+1: